llama-index-integrations/vector_stores/llama-index-vector-stores-qdrant/llama_index/vector_stores/qdrant/base.py

```diff
@@ -991,8 +993,17 @@ def parse_to_query_result(self, response: List[Any]) -> VectorStoreQueryResult:
 
         for point in response:
             payload = cast(Payload, point.payload)
+            vector = point.vector
+            embedding = None
+
+            if isinstance(vector, dict) and DENSE_VECTOR_NAME in vector:
```

Comment:
Technically the dense vector name could be `""` if it was created without hybrid enabled 

Can we add a quick unit test?